game.py

```diff
@@ -28,6 +27,13 @@
     "weather": ["sunny", "light_rain", "storm", "snow"]
 }
 
+def clear_term():
+    system_type = platform.system()
+
+    if system_type == "Windows":
+        os.system('cls')
+    else:
+        os.system('clear')
```

Comment:
needed to fix the script for windows when i got home and realized none of the clear commands worked. now it's a function and should work for any OS.